cmd/srcd-server/engine/engine.go

```diff
@@ -152,7 +154,11 @@ func (s *Server) parse(ctx context.Context, req *api.ParseRequest, log logf) (*a
 	return resp, nil
 }
 
-func createBbblfshd() error {
+func (s *Server) withWorkdirMounted(at string) docker.ConfigOption {
```

Comment:
wondering whether this function is actually needed
it's only called once and the body is simple enough I would just manually inline it